### PR TITLE
Form submission returns a Gemini generated workout.

### LINKED ISSUE #6: User can rearrange exercises in the workout

### LINKED ISSUE #39: Add testing for GeminiService

### PR DESCRIPTION
**Pull request by danxguerrero**
Closes #39, closes #6

### CODE REVIEW

File: src/routes/form/Form.jsx [hunk 1]
```diff
@@ -1,16 +1,95 @@
...
+    try {
+      const accessToken = await getAccessTokenSilently({
+        authorizationParams: {
+          audience: 'http://localhost:3001'
+        }
+      })
+
+      const workout = await fetchNewGeminiWorkout(accessToken, user, focusArea, type, level)
+      // TODO: delete this console.log before pushing
+      console.log("Navigating away")
```
Comment (Aaron-G-Sanchez) [inline src/routes/form/Form.jsx:42]: Nit:😬

File: src/routes/form/Form.jsx [hunk 2]
```diff
@@ -1,16 +1,95 @@
...
+      navigate('/workout', { state: {workout: workout}})
+    } catch (error) {
+      console.log("error: ", error)
+    }
+  }
+
   return (
     <>
       <main>
```
Comment (Aaron-G-Sanchez) [inline src/routes/form/Form.jsx:51]: I know this was added before you made your changes, but App.jsx should be rendering a main tag so this is redundant now.

**Review by Aaron-G-Sanchez — CHANGES_REQUESTED** [review]
LGTM! Just quick minor changes

File: src/routes/form/Form.jsx (hunk 2 above)
Comment (danxguerrero) [reply src/routes/form/Form.jsx]: Done.

File: src/routes/form/Form.jsx (hunk 1 above)
Comment (danxguerrero) [reply src/routes/form/Form.jsx]: Done.

**Review by Aaron-G-Sanchez — APPROVED** [review]
👍🏼

**Pull request merged**